app/controllers/lead_providers/confirm_schools_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module LeadProviders
+  class ConfirmSchoolsController < ApplicationController
+    before_action :load_form
+
+    def show
+      @schools = School.where(id: @form.school_ids).order(:name)
+      @delivery_partner = DeliveryPartner.find(@form.delivery_partner_id)
```

Comment:
Maybe we want the form to have `delivery_partner` and `schools` methods?

Reply:
I am not certain about this as it couples form to both models. I think i prefer the form to be just a simple data structure.